src/patterns/components/lists/angular/code/bare-list.hbs

```diff
@@ -1,11 +1,8 @@
-<sprk-unordered-list listType="bare">
-  <sprk-list-item>
-    Bare List Item
+<sprk-unordered-list listType="bare" idString="unordered-list-1">
+  <sprk-list-item idString="list-item-1">
+    cats
   </sprk-list-item>
-  <sprk-list-item>
-    Bare List Item
-  </sprk-list-item>
-  <sprk-list-item>
-    Bare List Item
+  <sprk-list-item idString="list-item-2">
+    dogs
```

Comment:
These need to match the documentation site. So, that when users copy the code blocks they get what they are seeing in the docs. This needs to stay as "Bare List Item".